src/azul/service/repository_controller.py

```diff
@@ -35,11 +39,66 @@ def service(self):
     def repository_plugin(cls, catalog: CatalogName) -> RepositoryPlugin:
         return RepositoryPlugin.load(catalog).create(catalog)
 
+    @classmethod
+    def parse_range_request_header(cls,
+                                   headers: Mapping[str, str]
```

Comment:
This should have just the header value as a string.

And it should be protected.